themes/default/views/editor/objects/summary_html.php

```diff
@@ -38,6 +38,12 @@
 <?php
 	if ($vs_display_select_html = $t_display->getBundleDisplaysAsHTMLSelect('display_id', array('onchange' => 'jQuery("#caSummaryDisplaySelectorForm").submit();',  'class' => 'searchFormSelector'), array('table' => $t_item->tableNum(), 'value' => $t_display->getPrimaryKey(), 'access' => __CA_BUNDLE_DISPLAY_READ_ACCESS__, 'user_id' => $this->request->getUserID(), 'restrictToTypes' => array($t_item->getTypeID())))) {
 ?>
+		<div id="toggleCollapsedButton">
```

Comment:
Lol yeah, of course. I always meant to apply that to the other files and possibly put it in a helper and then totally neglected to do that. I'm happy to add it to this PR if you prefer.